core/src/components/item/item.tsx

```diff
@@ -62,7 +62,7 @@ export class Item implements ComponentInterface, AnchorInterface, ButtonInterfac
   /**
    * The icon to use when `detail` is set to `true`.
    */
-  @Prop() detailIcon = chevronForward;
+  @Prop() detailIcon?: string | null;
```

Comment:
Technically, I didn't change the default detail icon, I've added a custom function that resolves the icon.
The reason behind this decision is that when a developer chooses the `detailIcon={chevronForward}` it won't flip automatically.

Reply:
We're still changing the type of the prop, though, since it can now accept `null`. This will also change the auto-generated docs, which pull in the default assigned here.

If the dev assigns their own icon, they can use custom icons for each case, something like: `detailIcon={document.dir === 'rtl' ? chevronBack : chevronForward}`